java/examples/README.md

```diff
@@ -1,8 +1,55 @@
-Building and Running
---------------------
+# CuVS Java API Examples
 
-Make sure to have JDK 22 and Maven 3.9.6+.
+This maven project contains simple examples for CAGRA, HNSW, and Bruteforce algorithms.
 
-    mvn clean compile assembly:single
+## Prerequisites
+- All the prerequisites in the CuVS Java API readme
+- libcuvs libraries and Java API should already be built
+- In this directory do `mvn package`
 
-    java --enable-native-access=ALL-UNNAMED -jar ./target/cagra-sample-1.0-SNAPSHOT-jar-with-dependencies.jar
+## Run Examples
+
+### CAGRA Example
+Doing the following in the current directory:
+```
+java --enable-native-access=ALL-UNNAMED -cp target/cuvs-java-examples-25.02.0.jar:$HOME/.m2/repository/com/nvidia/cuvs/cuvs-java/25.02.0/cuvs-java-25.02.0.jar com.nvidia.cuvs.examples.CagraExample
+```
+Should output the following (with different timestamps):
```

Comment:
This seems unecessary- I would just provide the command-line to run the example. Let the unit tests verify the results.